panel/browser-panel.cpp

```diff
@@ -202,11 +205,16 @@ void QCefWidgetInternal::closeBrowser()
 		 * So, instead, before closing the browser, we need to decouple
 		 * the browser from the widget.  To do this, we hide it, then
 		 * remove its parent. */
+#ifdef _WIN32
 		HWND hwnd = (HWND)cefBrowser->GetHost()->GetWindowHandle();
 		if (hwnd) {
 			ShowWindow(hwnd, SW_HIDE);
 			SetParent(hwnd, nullptr);
 		}
+#elif __APPLE__
+		// felt hacky, might delete later
+		id view = (id)cefBrowser->GetHost()->GetWindowHandle();
+		objc_msgSend(view, sel_getUid("removeFromSuperview"));
```

Comment:
Calling `objc_msgSend` directly is discouraged and Xcode 11 refuses to compile, this works however:

```
	id view = (id)cefBrowser->GetHost()->GetWindowHandle();
        SEL removeFromViewSel = sel_getUid("removeFromSuperview");
        
        typedef id(*remove_panel_func)(id, SEL);
        remove_panel_func removePanel = (remove_panel_func)objc_msgSend;
        
        removePanel(view, removeFromViewSel);
```